.env.example

```
@@ -4,3 +4,4 @@ DB_PORT=3306
 DB_USER=example-user
 DB_PWD=example-password
 DB_NAME=sexta_eh_nois
+JWT_SECRET=L50lneVpLCc4N4kDg2RMqXWioKs2F1xgwGAe8bI49w9roqkQFB
```

```suggestion
JWT_SECRET=secret-md5-hash
```
Geralmente é bom colocar exemplos reais, mas nesse caso, acredito que é melhor não deixar.